api/src/identity-access-management/application/anonymization/anonymization.admin.controller.js

```diff
@@ -0,0 +1,21 @@
+import { GARAnonymizationParser } from '../../domain/services/GARAnonymizationParser.js';
+import { usecases } from '../../domain/usecases/index.js';
+import { anonymizeGarResultSerializer } from '../../infrastructure/serializers/jsonapi/anonymize-gar-result.serializer.js';
+
+/**
+ * @param request
+ * @param h
+ * @return {Promise<*>}
+ */
+async function anonymizeGARData(request, h) {
```

Comment:
Pour respecter les standards de nommage, utiliser plutôt le nommage suivant : 
```suggestion
async function anonymizeGarData(request, h) {
```